release-notes/6.0/supported-os.json

```diff
@@ -393,11 +392,9 @@
                         "x86"
                     ],
                     "supported-versions": [
-                        "23H2",
```

Comment:
According to https://mcr.microsoft.com/en-us/product/windows/servercore and https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/base-image-lifecycle there was never a release of 23H2 or 2012.

Reply:
And besides https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/base-image-lifecycle stating that 1607 is only Semi-Anual and released on Nano Server, whereas 2016 is the only Long-Term on Server Core:
![image](https://github.com/user-attachments/assets/81db3ddd-0652-474c-9df3-af394038c289)

https://mcr.microsoft.com/en-us/product/windows/servercore shows that it's instead handled like the Server-Core case with 2019 and 1809:
![image](https://github.com/user-attachments/assets/0071ecee-07fd-4552-8dce-2f0ae558095b)

Reply:
Server 2012: https://learn.microsoft.com/en-us/lifecycle/products/windows-server-2012

Reply:
But since there are no images available, there is nothing to have support for 🤔

Reply:
Ok, I found https://learn.microsoft.com/en-us/previous-versions/windows/desktop/legacy/hh846313(v=vs.85) so I missed to add 2012 to EoL.date.

Will have a PR there tomorrow and fix it here now.